pom.xml

```diff
@@ -58,6 +58,11 @@
       <artifactId>token-macro</artifactId>
       <version>1.10</version>
     </dependency>
+    <dependency>
+      <groupId>com.github.docker-java</groupId>
+      <artifactId>docker-java</artifactId>
+      <version>1.3.0</version>
```

Comment:
> This will cause conflicts with docker-plugin if we pull it in ourselves without adding the classloader masking tricks

Please remember that normal Java calls to APIs work fine even if multiple plugins bundle the same or different versions of the same (i.e., unshaded) library. Problems only arise under special conditions:

1. Plugins A1 and A2 both bundle the library L, and also export its types from their own APIs, and plugin B depends on _both_ A1 and A2, and makes a call on A1 and passes the result to a call on A2.
1. The library attempts to load classes from itself reflectively (`Class.forName`) using `Thread.contextClassLoader`, and the application defines this CCL to include classes from all plugins (which Jenkins does not, despite having an `uberClassLoader`).
1. Some non-Java code, such as a Jelly or Groovy view, tries to load library classes directly via `uberClassLoader`.
1. The library is bundled in Jenkins core.